src/ontology/mondo-edit.obo

```diff
@@ -114079,13 +114079,32 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/10642/cerebe
 id: MONDO:0006132
 name: cervical adenoid basal carcinoma
 def: "A rare low grade carcinoma that arises from the cervix. It is characterized by the presence of nests of basaloid cells with focal glandular formations." [NCIT:P378]
+subset: gard_rare {source="GARD:20499"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:213828"}
+subset: orphanet_rare {source="Orphanet:213828"}
+subset: rare
+synonym: "adenoid basal carcinoma of the cervix uteri" EXACT [MONDO:0016287]
 synonym: "cervical adenoid basal cancer" EXACT [NCIT:C40213]
+synonym: "cervical adenoid basal carcinoma" NARROW [Orphanet:213828]
+synonym: "skin adenoid basal cell carcinoma of uterine cervix" EXACT [MONDO:design_pattern]
+synonym: "uterine cervix skin adenoid basal cell carcinoma" EXACT [MONDO:patterns/location]
 xref: DOID:6428 {source="MONDO:equivalentTo"}
 xref: EFO:1000160 {source="MONDO:equivalentTo"}
+xref: GARD:20499 {source="Orphanet:213828"}
+xref: ICD10CM:C53.0 {source="MONDO:relatedTo", source="Orphanet:213828/btnt", source="Orphanet:213828"}
+xref: ICD10CM:C53.1 {source="Orphanet:213828/btnt", source="Orphanet:213828"}
+xref: ICD10CM:C53.8 {source="Orphanet:213828/btnt", source="Orphanet:213828"}
 xref: NCIT:C40213 {source="DOID:6428", source="MONDO:equivalentTo", source="EFO:1000160"}
 xref: ONCOTREE:CABC {source="MONDO:equivalentTo"}
+xref: Orphanet:213828 {source="MONDO:equivalentTo"}
 xref: UMLS:C1516403 {source="DOID:6428", source="MONDO:equivalentTo", source="NCIT:C40213"}
+is_a: MONDO:0002974 {source="MONDO:Redundant", source="Orphanet:213828"} ! cervical cancer
```

Comment:
I don't think this `is_a` is needed since this term already has the parent `cervical carcinoma`, which is already a direct child of `cervical cancer` so this seems redundant.